epregressions/tk_window.py

```diff
@@ -812,6 +838,8 @@ def set_gui_status_for_run(self, is_running: bool):
         self.remove_idf_from_active_button.configure(state=run_button_state)
         self.run_period_option_menu.configure(state=run_button_state)
         self.reporting_frequency_option_menu.configure(state=run_button_state)
+        self.force_output_sql_option_menu.configure(state=run_button_state)
+        self.force_output_sql_unitconv_option_menu.configure(state=run_button_state)
```

Comment:
disable when run is launched. 

I think ideally I should also just enable the force_output_sql_unitconv_option_menu only when force_output_sql_option_menu is something else than NOFORCE